Source/Scene/Model.js

```diff
@@ -1453,6 +1453,103 @@ define([
+                    // count property.
+                    //
+                    // https://github.com/KhronosGroup/glTF/issues/258
+
+                    var materialId = primitive.material;
+                    var material = gltf.materials[materialId];
+                    var technique = gltf.techniques[material.technique];
+                    
+                    var programId = technique.program;
+                    if (programId === programName) {
+                        var primitiveAttributes = primitive.attributes;
+                        var attributeVarName;
+                        for (var attributeSemantic in primitiveAttributes) {
+                            if (primitiveAttributes.hasOwnProperty(attributeSemantic)) {
+                                attributeVarName = undefined;
+                                for (var attributeName in technique.parameters) {
+                                    if (technique.parameters.hasOwnProperty(attributeName)) {
+                                        if (defined(attributeVarName)) {
+                                            break;
+                                        }
+                                        if (technique.parameters[attributeName].semantic === attributeSemantic) {
+                                            for (var varName in technique.attributes) {
+                                                if (technique.attributes.hasOwnProperty(varName)) {
+                                                    if (technique.attributes[varName] === attributeName) {
+                                                        attributeVarName = varName;
+                                                        break;
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                                // Skip if the attribute is not used by the material, e.g., because the asset was exported
+                                // with an attribute that wasn't used and the asset wasn't optimized.
+                                if (defined(attributeVarName)) {
+                                    var a = accessors[primitiveAttributes[attributeSemantic]];
+                                    if (a.hasOwnProperty('extensions')) {
+                                        var extensions = a.extensions;
+                                        if (extensions.hasOwnProperty('WEB3D_quantized_attributes')) {
+                                            var quantizedAttributes = extensions.WEB3D_quantized_attributes;
+                                            var decodeMatrix = quantizedAttributes.decodeMatrix;
+                                            decodeMatrix = decodeMatrix.map(function(val) {
```

Comment:
We don't change datatypes of existing variables like this, e.g., from number to string.

Also, it may not matter in practice, but I would use the max allowed value of `20` for the parameter to `toFixed`.